src/main/java/in/projecteka/consentmanager/link/ClientErrorExceptionHandler.java

```diff
@@ -44,6 +45,11 @@ public ClientErrorExceptionHandler(
             bodyInserter = BodyInserters.fromValue(((ClientError) error).getError());
         }
 
+        if (error instanceof DbOperationError) {
+            status = ((DbOperationError) error).getHttpStatus();
```

Comment:
Can you add log statement, should not take much time? log error irrespective type in this class.